src/twisted/internet/_sslverify.py

```diff
@@ -1456,8 +1551,12 @@ def __init__(self,
             ciphers for the current platform.
 
         @raise TypeError: if C{trustRoot} is passed in combination with
-            C{caCert}, C{verify}, or C{requireCertificate}.  Please prefer
+            C{caCert}, C{verify}, or C{requireCertificate}. Please prefer
```

Comment:
https://github.com/glyph/python-docstring-mode will format this correctly automatically, btw.